trinity/protocol/bcc/commands.py

```diff
@@ -0,0 +1,36 @@
+from rlp import sedes
+
+from p2p.protocol import (
+    Command,
+)
+
+from trinity.rlp.sedes import HashOrNumber
+from eth.beacon.types.blocks import BaseBeaconBlock
+from eth.beacon.types.attestation_records import AttestationRecord
+
+
+class Status(Command):
+    _cmd_id = 0
+    structure = [
+        ('protocol_version', sedes.big_endian_int),
+        ('network_id', sedes.big_endian_int),
+        ('best_hash', sedes.binary),
+    ]
+
+
+class GetBeaconBlocks(Command):
+    _cmd_id = 1
+    structure = [
+        ('block_number_or_hash', HashOrNumber()),
```

Comment:
Note that it's actually `block slot`. Maybe:
```suggestion
        ('block_slot_or_hash', HashOrNumber()),
```